tests/e2e/CreateExam.spec.ts

```diff
@@ -0,0 +1,287 @@
+        cy.get("button[id='examDate_minutes']").click();
+        cy.get(`li[id='examDate_minutesItem-${numberZeroPad(examDate.getMinutes())}']`).click();
+
+        //set deadlines for admission and drop
+        cy.get("button[id='admitUntilDate_day']").click();
+        cy.get(`li[id='admitUntilDate_dayItem-${admittableUntilDate.getDay()}']`).click();
+        cy.get("button[id='admitUntilDate_month']").click();
+        cy.get(`li[id='admitUntilDate_monthItem-${months[admittableUntilDate.getMonth() - 1]}']`).click();
+        cy.get("button[id='admitUntilDate_year']").click();
+        cy.get(`li[id='admitUntilDate_dayItem-${admittableUntilDate.getFullYear()}']`).click();
+        cy.get("button[id=admitUntilDate_hour']").click();
+        cy.get(`li[id='admitUntilDate_hourItem-${numberZeroPad(admittableUntilDate.getHours())}']`).click();
+        cy.get("button[id='admitUntilDate_minutes']").click();
+        cy.get(`li[id='admitUntilDate_minutesItem-${numberZeroPad(admittableUntilDate.getMinutes())}']`).click();
+
+        cy.get("button[id='dropUntilDate_day']").click();
+        cy.get(`li[id='dropUntilDate_dayItem-${droppableUntilDate.getDay()}']`).click();
+        cy.get("button[id='dropUntilDate_month']").click();
+        cy.get(`li[id='dropUntilDate_monthItem-${months[droppableUntilDate.getMonth() - 1]}']`).click();
+        cy.get("button[id='dropUntilDate_year']").click();
+        cy.get(`li[id='dropUntilDate_dayItem-${droppableUntilDate.getFullYear()}']`).click();
+        cy.get("button[id='dropUntilDate_hour']").click();
+        cy.get(`li[id='dropUntilDate_hourItem-${numberZeroPad(droppableUntilDate.getHours())}']`).click();
+        cy.get("button[id='dropUntilDate_minutes']").click();
+        cy.get(`li[id='dropUntilDate_minutesItem-${numberZeroPad(droppableUntilDate.getMinutes())}']`).click();
+
+        cy.get("button[id='createExam']").click();
+    });
+
+    it("Encryption modal should be shown", () => {
+        cy.get("#modal-wrapper").should("exist");
+        cy.get("div").contains(
+            "Please choose a password to encrypt your private key, so it can be securely stored on our servers. Ensure that you do not lose this password as it cannot be restored."
+        );
+    });
+
+    it("Canceling the modal does not result in error", () => {
+        cy.get("button[id='encryptPrivateKeyModalCancel']").click();
+        cy.get("button[id='encryptPrivateKeyModalCancel']").should("not.be.visible");
+        cy.get("button[id='createExam']").click();
+    });
+
+    it("Enter encryption password to proceed", () => {
+        const encryptionPassword = studentAuthUser.password;
+        cy.get("input[id='enterEncryptionPassword']").clear().type(encryptionPassword);
+        cy.get("input[id='confirmEncryptionPassword']").clear().type(encryptionPassword);
+        cy.get("button[id='encryptPrivateKeyModalConfirm']").click();
+    });
+
+    //TODO What happens if we cancel here?
```

Comment:
This should be clarified and then implemented or moved to an issue.

Reply:
I removed that comment as it was a remainder from an older state of the application.